src/EditorFeatures/Core/Implementation/Organizing/OrganizeDocumentCommandHandler.cs

```diff
@@ -106,6 +122,20 @@ public bool ExecuteCommand(SortAndRemoveUnnecessaryImportsCommandArgs args, Comm
             return true;
         }
 
+        private void SortImports(ITextBuffer subjectBuffer, IUIThreadOperationContext operationContext)
+        {
+            var cancellationToken = operationContext.UserCancellationToken;
+            var document = subjectBuffer.CurrentSnapshot.GetFullyLoadedOpenDocumentInCurrentContextWithChangesAsync(operationContext).WaitAndGetResult(cancellationToken);
```

Comment:
Why the "fully loaded" in this case? For unused we need semantics, isn't sorting syntactic only?